include/onnxruntime/core/graph/graph.h

```diff
@@ -840,6 +845,11 @@ class Graph {
   Node& AddNode(const ONNX_NAMESPACE::NodeProto& node_proto,
                 const ArgNameToTypeMap& name_to_type);
 
+  common::Status DoInputsHaveFP16();
```

Comment:
>DoInputsHaveFP16 [](start = 17, length = 16)

should be const.

Reply:
This is the helper. I made the public method const.

---
In reply to: [382315089](https://github.com/microsoft/onnxruntime/pull/3061#discussion_r382315089) [](ancestors = 382315089)